eng/codegen_to_sdk_config.json

```
@@ -0,0 +1,18 @@
+{
+  "init": {
+      "initScript": {
+        "path": ".scripts/automation_init.sh"
+      }
+    },
+  "generateAndBuild": {
+      "generateAndBuildScript": {
+        "path": ".scripts/automation_generate.sh",
+        "stdout": {
+          "storeLogByFilter": "Error: |error |throw|exception"
+        },
+        "stderr": {
+          "storeLogByFilter": "Error: |error |throw|exception"
+        }
+      }
+  }
```

The json indentation is slightly off here:

```suggestion
  "init": {
    "initScript": {
      "path": ".scripts/automation_init.sh"
    }
  },
  "generateAndBuild": {
    "generateAndBuildScript": {
      "path": ".scripts/automation_generate.sh",
      "stdout": {
        "storeLogByFilter": "Error: |error |throw|exception"
      },
      "stderr": {
        "storeLogByFilter": "Error: |error |throw|exception"
      }
    }
  }
```